redis/server/install.sls

```diff
@@ -0,0 +1,104 @@
+include:
+  - redis.common
+
+{% from "redis/map.jinja" import redis_settings as r with context %}
+
+    {%- if r.install_from == 'source' %}
```

Comment:
Why indentation does not start from column 0?

The two branches of this if may be better in separate `.sls` files, for example:
- `source.sls`
- `package.sls`

But this would be better done in an update to adopt the component style with `package`, `service`, etc.

Reply:
The indentation is my style.  Jinja from/includes can start at column 0, but Jinja conditionals must be indented to differentiate  A&B languages (sls, jinja).   I agree and might look at that in the future because that what I prefer also but I just wanted to restrain myself from too many changes